application/src/main/java/org/thingsboard/server/actors/calculatedField/CalculatedFieldManagerMessageProcessor.java

```diff
@@ -109,7 +117,7 @@ public void onFieldInitMsg(CalculatedFieldInitMsg msg) throws CalculatedFieldExc
     }
 
     public void onLinkInitMsg(CalculatedFieldLinkInitMsg msg) {
-        log.info("[{}] Processing CF link init message for entity [{}].", msg.getLink().getCalculatedFieldId(), msg.getLink().getEntityId());
+        log.debug("[{}] Processing CF link init message for entity [{}].", msg.getLink().getCalculatedFieldId(), msg.getLink().getEntityId());
```

Comment:
Please also fix this one
![image](https://github.com/user-attachments/assets/c2db488c-8bcf-4490-b19d-9bb192a7e3c7)